CHANGELOG.md

```diff
@@ -2,6 +2,68 @@
 
 All notable changes are documented in this file. Release numbers follow [Semantic Versioning](http://semver.org)
 
+## 0.9.0.Beta2
+December 19th, 2018 [Detailed release notes](https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12317320&version=12339976)
+
+### New features since 0.9.0.Beta1
+
+* Add support for Oracle 11g [DBZ-954](https://issues.jboss.org/browse/DBZ-954)
+* UnwrapFromMongoDbEnvelope refactor [DBZ-1020](https://issues.jboss.org/browse/DBZ-1020)
+* Add option for dropping deletes and tombstone events to MongoDB struct recreation SMT [DBZ-563](https://issues.jboss.org/browse/DBZ-563)
+* Expose "snapshot.delay.ms" option for all connectors [DBZ-966](https://issues.jboss.org/browse/DBZ-966)
+* Convey original operation type when using flattening SMTs [DBZ-971](https://issues.jboss.org/browse/DBZ-971)
+* Provide last event and captured tables in metrics [DBZ-978](https://issues.jboss.org/browse/DBZ-978)
+* Skip MySQL BinLog Event in case of Invalid Cell Values [DBZ-1010](https://issues.jboss.org/browse/DBZ-1010)
+
+
+### Breaking changes since 0.9.0.Beta1
```

Comment:
Hello @jpechane 
I'd like to point out there's a breaking change on the mongo transform https://github.com/debezium/debezium.github.io/pull/228/files#diff-ad2db9e5c7d0dda9de74c7ddc4bbef4bR19